frontend/staff-dashboard/src/utils.ts

```diff
@@ -16,3 +16,8 @@ export const formatDiscount = (discount: IDiscount) => {
     }
     return formattedDiscount;
 }
+
+
+export const formatIncome = (income: string, currency: string) => {
+    return currency + " " + parseFloat(income).toLocaleString(undefined, {maximumFractionDigits: 2})
```

Comment:
Can probably set this to `en-US` for the locale and then use the `currency` options here (with the currency passed) to format these like a money string (e.g. $1,234.56 or €2 345,99).

Reply:
Yeah, this makes sense. I wasn't sure if the currency would be added correctly from the admin site. Initially, I was about to go this way but then just backed off due to this reason.